lib/customer.rb

```diff
@@ -0,0 +1,43 @@
+# require './order'
+# require './menu'
+
+class Customer
+
+attr_accessor :phone_number, :choices, :total_estimate, :menu
+
+  def initialize phone_number
+    @phone_number = phone_number
+  end
+
+  def request menu
+    @menu = menu.menu_items
+  end
+  
+  def real_total
+    choices.inject(0) { |sum, x| sum + menu[x] }
+  end
+
+  def validate_order?
+   return false if real_total != total_estimate
+   true
+  end
```

Comment:
Does these need to be a public method? I feel like it could be private without too much fuss